source/includes/steps-open-new-playground.yaml

```diff
@@ -0,0 +1,34 @@
+title: Open the Visual Studio Code :guilabel:`Command Palette`.
+level: 4
+ref: open-cmd-palette
+content: |
+    In Visual Studio Code, press either
+
+    - :guilabel:`Control + Shift + P` or
+
+    - :guilabel:`Command + Shift + P`.
+
+    The :guilabel:`Command Palette` provides quick access to commands
+    and keyboard shortcuts.
+---
+title: Find and select the "Create MongoDB Playground" command.
```

Comment:
Should this be "Find and run the "Create MongoDB Playground" command."?

Reply:
It could be. You're selecting a command from a dropdown list, but "run" is probably a better word choice.